### DISCUSSION TITLE
New area for L & R buttons

### DISCUSSION THREAD

**Discussion by PiixelNerd** [rafaelvcaetano/melonDS-android#1007] [Ideas]
See a possible change to the shoulder buttons. It can make it easier to use when trying to touch them by having more contact area horizontally. The remake of the 2 buttons are in the image, if you want to use it, you can use it.

![MelonDS new LR area](https://user-images.githubusercontent.com/93597875/202854606-3eef09f5-47bf-475e-a190-2628e6182156.png)

**Comment by rafaelvcaetano** [comment]
These buttons definitely need a new touch. I'll take a look see if the size that you chose makes sense or if it needs a small adjustment. But I appreciate your suggestion a lot :)
